tests/collection/collection-layout.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { test } from '../fixtures';
+
+import { LayoutViewModeLocator, SearchOption } from '../models';
+
+test.beforeEach(async ({ collectionPage }) => {
+  test.info().annotations.push({
+    type: 'Test',
+    description: 'Collection MDS search every each test',
+  });
+
+  await test.step(`Select "Search metadata"`, async () => {
+    await collectionPage.searchPage.clickSearchInputOption(
+      SearchOption.METADATA,
+    );
+  });
+
+  await test.step(`Query for "radio"`, async () => {
+    await collectionPage.searchPage.queryFor('radio');
+  });
+
+  await test.step(`Total result count is displayed`, async () => {
+    await collectionPage.collectionFacets.checkResultCount();
+  });
+});
```

Comment:
unsure what we are testing for here - looks like these test steps are in the `beforeEach` block? 
q: what does ``Query for "radio"` mean? what's the step we are trying to validate here?

Reply:
it's a test step that we need to do a metadata search in each tests